tsl/src/hypertable.c

```diff
@@ -162,9 +162,9 @@ hypertable_get_and_validate_data_nodes(ArrayType *nodearr)
 
 	if (num_data_nodes == 0)
 		ereport(ERROR,
-				(errcode(ERRCODE_TS_NO_DATA_NODES),
+				(errcode(ERRCODE_TS_INSUFFICIENT_NUM_DATA_NODES),
 				 errmsg("no data nodes can be assigned to the hypertable"),
-				 errhint("Add data nodes using the add_data_node() function.")));
+				 errhint("Add new data nodes to the database.")));
```

Comment:
Nit. Using "new" suggest that there are already old data nodes, which is not the case.
```suggestion
				 errhint("Add data nodes to the database.")));
```